be/src/olap/snapshot_manager.cpp

```diff
@@ -598,11 +558,16 @@ OLAPStatus SnapshotManager::_convert_beta_rowsets_to_alpha(const TabletMetaShare
                 res = OLAP_ERR_INIT_FAILED;
                 break;
             }
-            new_rowsets->push_back(new_rowset);
+            LOG(INFO) << "convert beta rowset:" << rowset_meta->rowset_id()
+                    << " to alpha rowset:" << new_rowset->rowset_id();
+            new_rowsets.push_back(new_rowset);
         } else {
-            new_rowsets->push_back(rowset_meta);
+            new_rowsets.push_back(rowset_meta);
         }
     }
+    if (res == OLAP_SUCCESS && modified) {
```

Comment:
if request.__isset.missing_version is empty, you should call new_tablet_meta->revise_rs_metas()

Reply:
if modified is false, no need to call revise

Reply:
ok